charts/opentelemetry-collector/values.yaml

```diff
@@ -207,6 +207,10 @@ agentCollector:
 
   podLabels: {}
 
+  # Host networking requested for this pod. Use the host's network namespace.
+  # If this option is set, the ports that will be used must be specified.
```

Comment:
Can you clarify what you mean by "ports that will be used must be specified."? I thought if hostNetwork was true that every port exposed on the pod gets exposed on the hostNetwork.  Is there extra specificity that has to happen?

Reply:
The `ports` blocks for `containers` on a `Pod` are optional when forwarding traffic from a `Service` but required when using `hostNetwork: true`

I copied the text from the Kubernetes API docs... Maybe we need less documentation here in this case? Since we are always setting the ports on the containers?

What do you think?

Reply:
Ya that works.  If the second comment is removed I think the setting will still be understood.